deepmd/dpmodel/descriptor/dpa3.py

```diff
@@ -0,0 +1,647 @@
+            Maximally possible number of selected edge neighbors.
+        a_rcut : float, optional
+            The angle cut-off radius.
+        a_rcut_smth : float, optional
+            Where to start smoothing for angle. For example the 1/r term is smoothed from rcut to rcut_smth.
+        a_sel : int, optional
+            Maximally possible number of selected angle neighbors.
+        a_compress_rate : int, optional
+            The compression rate for angular messages. The default value is 0, indicating no compression.
+            If a non-zero integer c is provided, the node and edge dimensions will be compressed
+            to a_dim/c and a_dim/2c, respectively, within the angular message.
+        a_compress_e_rate : int, optional
+            The extra compression rate for edge in angular message compression. The default value is 1.
+            When using angular message compression with a_compress_rate c and a_compress_e_rate c_e,
+            the edge dimension will be compressed to (c_e * a_dim / 2c) within the angular message.
+        a_compress_use_split : bool, optional
+            Whether to split first sub-vectors instead of linear mapping during angular message compression.
+            The default value is False.
+        n_multi_edge_message : int, optional
+            The head number of multiple edge messages to update node feature.
+            Default is 1, indicating one head edge message.
+        axis_neuron : int, optional
+            The number of dimension of submatrix in the symmetrization ops.
+        update_angle : bool, optional
+            Where to update the angle rep. If not, only node and edge rep will be used.
+        update_style : str, optional
+            Style to update a representation.
+            Supported options are:
+            -'res_avg': Updates a rep `u` with: u = 1/\\sqrt{n+1} (u + u_1 + u_2 + ... + u_n)
+            -'res_incr': Updates a rep `u` with: u = u + 1/\\sqrt{n} (u_1 + u_2 + ... + u_n)
+            -'res_residual': Updates a rep `u` with: u = u + (r1*u_1 + r2*u_2 + ... + r3*u_n)
+            where `r1`, `r2` ... `r3` are residual weights defined by `update_residual`
+            and `update_residual_init`.
+        update_residual : float, optional
+            When update using residual mode, the initial std of residual vector weights.
+        update_residual_init : str, optional
+            When update using residual mode, the initialization mode of residual vector weights.
+        fix_stat_std : float, optional
+            If non-zero (default is 0.3), use this constant as the normalization standard deviation
+            instead of computing it from data statistics.
+        skip_stat : bool, optional
+            (Deprecated, kept only for compatibility.) This parameter is obsolete and will be removed.
+            If set to True, it forces fix_stat_std=0.3 for backward compatibility.
+            Transition to fix_stat_std parameter immediately.
```

Comment:
Note the relationship between `skip_stat` and `fix_stat_std`. I kept `skip_stat` for compatibility with already-trained models. We can remove `skip_stat` in the future.